src/output.h

```diff
@@ -142,6 +142,13 @@ nc_color msgtype_to_color( const game_message_type type, const bool bOldMsg = fa
  * color tags. For example `utf8_width("<color_red>text</color>")` would return 23, but
  * `utf8_width("<color_red>text</color>", true)` returns 4 (the length of "text").
  */
+
+/*@{*/
+/**
+ * Adds a color tag on the input string.
```

Comment:
Is this really necessary? What information does this comment add?

Anyway, you don't need the member grouping via `/** @{ */` (which you don't actually close).

Reply:
I'm just following the style of other comments in this file. You don't want it? Fine with me... I don't like to leave not commented declarations in headers, but I'll follow your call here.